File: team/src/main/java/de/frudisch/manager/team/domain/Team.java
package de.frudisch.manager.team.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;
import java.util.UUID;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Team {
    private UUID id;

    private List<UUID> members;

    private String name;

}

File: team-management/src/main/kotlin/de/frudisch.manager.team.management/domain/entity/Team.kt
package de.frudisch.manager.team.management.domain.entity

import java.util.*
import javax.persistence.*

@Entity
@Table(name = "T_TEAM")
data class Team(

        @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
        val uuid: UUID? = null,

        val name: String? = null,

        val organisationUUID : UUID? = null,

        val league: League? = null)